src/routes/ViewAmendmentRoute.js

```diff
@@ -106,6 +106,7 @@ class ViewAmendmentsRoute extends React.Component {
 
   urls = {
     editAmendment: this.props.stripes.hasPerm('ui-licenses.licenses.edit') && (() => `/licenses/${this.props.match.params.id}/amendments/${this.props.match.params.amendmentId}/edit${this.props.location.search}`),
+    licenseView: id => `/licenses/${id}`
```

Comment:
If you want to keep the `urls` object declared here as the source of truth for all the child components of this route, then 
you could import `urls` from utils and assign `licenseView: urls.licenseView`. That way we wont dont repeat the code multiple times.

Reply:
I did try this and it failed. I'll retry though

Reply:
Per slack, not doing this as both are unique to this route, neater to leave as is